care/facility/models/bed.py

```diff
@@ -40,7 +40,7 @@ def __str__(self):
         return self.name
 
     def validate(self) -> None:
-        if Bed.objects.filter(location=self.location, name=self.name).exists():
+        if Bed.objects.filter(location=self.location, name=self.name).exclude(pk=self.pk).exists():
```

Comment:
Finally, something that the todo app taught!

Reply:
😅